### ISSUE TITLE
Broadcast Email not being sent through Amazon SES after server restart

### ISSUE THREAD

**Issue by devabdo** [mautic/mautic#2702]
What type of report is this:

| Q | A |
| --- | --- |
| Bug report? |  |
| Feature request? |  |
| Enhancement? |  |
## Description:

Database broke down during the sending of a Broadcast Email. Server had been restarted and sending resumed, interface shows that all Emails are sent, yet Amazon SES sending stats shows only the first patch before restart sent.
## If a bug:

| Q | A |
| --- | --- |
| Mautic version | v2.2.0 |
| PHP version | 5.6.21 |
### Steps to reproduce:
1. send a broadcast email
2. restart server
3. resume sending broadcast email
4. while mautic says it's sent, emails are not actual sent.
### Log errors:

[2016-10-10 17:30:19] mautic.NOTICE: Symfony\Component\Console\Exception\RuntimeException: Too many arguments, expected arguments "command". (uncaught exception) at /home/egyprene/public_html/mautic/vendor/symfony/console/Input/ArgvInput.php line 181 while running console command `mautic:broadcasts:send` [] []
[2016-10-10 17:31:21] mautic.NOTICE: Swift_TransportException: Expected response code 250 but got code "", with message "" (uncaught exception) at /home/egyprene/public_html/mautic/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/AbstractSmtpTransport.php line 383 while running console command `mautic:emails:send` [] []

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by escopecz** [comment]
How do you _"resume sending broadcast email"_? Do you use the email queue?

**Comment by amagdy3** [comment]
After server restart the broadcast email showed "specific number" as pending - I clicked send and the screen showed sending to the number of remaining emails. By this time, sending was done through Email Queue / Spool.

Now, I have cloned the same email into a template Email, removed emojis, changed sending options into "Send Immediately" and triggered it through a campaign, Emails were sent. It took a lot of time but sent.

I have no idea why the error above happens and it did happen twice.
